k8s/namespaces/jenkins/patches/ptlsbox/cluster-00/jenkins.yaml

```diff
@@ -8,6 +8,9 @@ spec:
   values:
     controller:
       ingress:
+        annotations:
+          kubernetes.io/ingress.class: traefik
+          traefik.ingress.kubernetes.io/router.tls: "true"
```

Comment:
this should go in the main file not the sandbox specific one, the ingress class is already there:
https://github.com/hmcts/shared-services-flux/blob/master/k8s/namespaces/jenkins/jenkins.yaml#L270